google-home-community.groovy

```diff
@@ -386,9 +407,7 @@ def deviceTraitPreferences(deviceTrait) {
         title: "Preferences For ${GOOGLE_DEVICE_TRAITS[deviceTrait.type]} Trait",
         nextPage: "deviceTypePreferences"
     ) {
-        section {
-            "deviceTraitPreferences_${deviceTrait.type}"(deviceTrait)
-        }
```

Comment:
Good timing.  I was just trying to figure this error out after some people reported it in the forum thread.  I went ahead and pulled this change out to a separate commit (c3e51e1973c951477ed6da8a824d3aa839ecb51e) so it wouldn't be broken while you work on this PR.  Thanks for catching this; I was certainly confused.